### PR TITLE
#7300: Add support for sandboxed iframes (MV2 only)

### PR DESCRIPTION
**Pull request by fregante**
## What does this PR do?

- Fixes https://github.com/pixiebrix/pixiebrix-extension/issues/7300

## Test URLs

- Sandbox + regular iframe: https://more-ephiframe.vercel.app/Outer?sandbox=&iframe=./Full-sandbox&iframe=./No-sandbox
- Sandbox + regular srcdoc iframe: https://ephiframe.vercel.app/?sandbox=&srcdoc=%3Ch1%3ESandboxed%20srcdoc&srcdoc=%3Ch1%3EFree%20srcdoc

## Demo (MV2)

Notice the loading pixel in all 3 frames:

Screenshot 5

## Demo (MV3)

Notice the missing loading pixel in the first iframe. This is not yet supported.

MV3

## Blocked by

- https://github.com/awesome-webextension/webpack-target-webextension/pull/42

## Future work

- [ ] Add MV3 support for sandboxed iframes
  - Blocked by https://github.com/awesome-webextension/webpack-target-webextension/issues/43

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -27905,8 +27905,9 @@
     },
     "node_modules/webpack-target-webextension": {
       "version": "1.1.0",
-      "integrity": "sha512-ip7ljKtjRmXKjuLZLo8jzcQjV3nuVPI1U+AUcx6PGJXC/+7apt20wbSVfOKnXDhHhijzP44qe/1qSgR8C+HgZg==",
+      "resolved": "git+ssh://git@github.com/awesome-webextension/webpack-target-webextension.git#ccbae6251a28a5e797804b2b4393bf3357009373",
```
Comment (fregante) [inline package-lock.json:27908]: Since we use a lockfile, this PR isn't technically blocked by the upstream PR, it's already installable and mergeable.

**Review by fregante — COMMENTED** [review]
@twschiller wanna give this a go to see if it solves your issue?

**Review by twschiller — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (twschiller) [reply package-lock.json:27908]: It's in the `package.json` too. If it were just the lockfile, I'd be weary of merging

**Comment by twschiller** [comment]
I'm still running into issues with the sandboxed iframe

image

**Comment by twschiller** [comment]
Adding @grahamlangford as reviewer as FYI about why we're pinning to a PR commit

**Review by twschiller — APPROVED** [review]

**Pull request merged**